optimum/intel/openvino/modeling_base.py

```diff
@@ -279,6 +268,59 @@ def _from_pretrained(
             **kwargs,
         )
 
+    @classmethod
+    @add_start_docstrings(FROM_PRETRAINED_START_DOCSTRING)
+    def from_pretrained(
+        cls,
+        model_id: Union[str, Path],
+        export: bool = False,
+        force_download: bool = False,
+        use_auth_token: Optional[Union[bool, str]] = None,
+        token: Optional[Union[bool, str]] = None,
+        cache_dir: str = HUGGINGFACE_HUB_CACHE,
+        subfolder: str = "",
+        config: Optional[PretrainedConfig] = None,
+        local_files_only: bool = False,
+        trust_remote_code: bool = False,
+        revision: Optional[str] = None,
+        **kwargs,
+    ):
+        if use_auth_token is not None:
+            warnings.warn(
+                "The `use_auth_token` argument is deprecated and will be removed soon. Please use the `token` argument instead.",
+                FutureWarning,
+            )
+            if token is not None:
+                raise ValueError("You cannot use both `use_auth_token` and `token` arguments at the same time.")
+            token = use_auth_token
+
+        ov_files = _find_files_matching_pattern(
+            model_id,
+            pattern=r"(.*)?openvino(.*)?\_model.xml",
+            subfolder=subfolder,
+            use_auth_token=token,
+            revision=revision,
+        )
+        _export = len(ov_files) == 0
+        if _export ^ export:
```

Comment:
but what if I want force model conversion e.g. I already have converted model with previous openvino version and want to reconvert with new one and store both pt and ov model in the same directory?

Reply:
also how it works for pipelined models, when you need several parts to launch model? (e.g. for seq2seq model you have only openvino_decoder_model, but openvino_encoder_model missed, will it trigger model conversion, I guess not...

Reply:
> but what if I want force model conversion e.g. I already have converted model with previous openvino version and want to reconvert with new one and store both pt and ov model in the same directory?

you should force it with `export=True` in the `.from_pretrained()` API. To store both pt and ov at the same location you need to use AutoModeFor... and OVModelFor... and call `.save_pretrained()` for both.

Reply:
> you should force it with export=True in the .from_pretrained() API. To store both pt and ov at the same location you need to use AutoModeFor... and OVModelFor... and call .save_pretrained() for both.

yes, it should work this way in my understanding too, but looking on condition below, looks like not, because  in code below _export value (that True only if there is no converted models) provided in function for loading model. So force export=True will be ignored

Reply:
fixed in https://github.com/huggingface/optimum-intel/pull/825/commits/2d4fdbd6f6a406fd11ae20909cb39ec6ec618bd3